dashboard/src/pages/EventSchedule.vue

```
@@ -176,7 +176,7 @@ const showModifyScheduleItemDrawer = ref(false)
           variant="ghost"
           label="Go to Schedule Page"
           icon-right="arrow-up-right"
-          @click="redirectRoute(`schedule/${event.doc?.route}`, '_blank')"
+			@click="redirectRoute(`${event.doc?.route}/schedule`, '_blank')"
```

ESLint

[error] Replace `↹↹↹` with `··········`

(prettier/prettier)